apiFunctions.php

```diff
@@ -464,8 +464,28 @@ function expand_doi_with_dx($template, $doi) {
        $template->mark_inactive_doi($doi);
        return FALSE;
      }
+     $json = @json_decode($ris, TRUE);
+     if($json === FALSE) return FALSE;
      report_action("Querying dx.doi.org: doi:" . doi_link($doi));
-     $template->expand_by_RIS($ris, FALSE);
```

Comment:
As we used to use a function, is there a case for setting up a new function for the new code?

Reply:
Once the code is done, I will do that